src/buza/templates/buza/user_detail.html

```diff
@@ -1,15 +1,10 @@
-{% extends "base.html" %}
+{% extends 'base.html' %}
 
-{% block title %}View you account{% endblock %}
-{% block content %}
-<h1>{{user.firstname}} {{user.surname}}</h1>
-<br>
-<p>{{user.email}}</p>
-<p>{{profile.bio}}</p><br>
-<br>
-<p>{{profile.photo}}</p><br>
-<p>{{profile.school}}</p><br>
-<p>{{profile.grade}}</p><br>
+{% block title %}{{ user.get_full_name }} - {{ block.super }}{% endblock %}
```

Comment:
so that's how you override the block form a parent template

Reply:
Right, `{{ block.super }}` lets you insert the parent block, to extend it, which is very useful for things like titles. :)